UnityProject/Assets/Scripts/Messages/Client/ClientMessage.cs

```diff
@@ -32,11 +34,9 @@ public static void SendUnreliable(T msg)
 			NetworkClient.Send(msg, 1);
 		}
 
-		internal bool IsFromAdmin()
+		internal bool IsFromAdmin(string PermissionCode)
```

Comment:
```suggestion
		internal bool HasPermission(string PermissionCode)
```

Reply:
actually, the permissions code should already have a ``HasPermission`` method

Reply:
this has extra logging on